src/v/cluster_link/replication/deps.h

```diff
@@ -31,20 +31,20 @@ class data_sink {
     virtual kafka::offset last_replicated_offset() const = 0;
 
     virtual raft::replicate_stages replicate(
-      chunked_vector<model::record_batch> batches,
-      model::timeout_clock::duration timeout,
+      chunked_vector<::model::record_batch> batches,
```

Comment:
in retrospect, I think having a module-scoped `namespace model` is probably more trouble than it is worth.